.seed-config.yaml

```diff
@@ -0,0 +1,108 @@
+project:
+  name: "Godocit"
+  state: "unstable"
+  image: "/blob/master/logo.svg"
```

Comment:
Let's use the full image URL based on github that way it works reliably even in other viewers.